src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -405,10 +405,14 @@ ucs_status_t uct_rc_mlx5_ep_flush(uct_ep_h tl_ep, unsigned flags,
     return UCS_INPROGRESS;
 }
 
-ucs_status_t uct_rc_mlx5_ep_fc_ctrl(uct_rc_ep_t *rc_ep)
+ucs_status_t uct_rc_mlx5_ep_fc_ctrl(uct_ep_t *tl_ep, unsigned op, void *args)
```

Comment:
In case of DC we will pass DC specific information needed to send a grant (which will be stored in prvi field of rc_fc request). In case of RC we do not need it.